__tests__/app.test.js

```diff
@@ -160,3 +160,23 @@ describe("/api/reviews/:review_id", () => {
     });
   });
 });
+
+describe("/api/users", () => {
+  describe("GET", () => {
+    test("status 200: should return array of user objects  ", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body).toHaveProperty("users");
+          const { users } = body;
+          expect(users).toBeInstanceOf(Array);
+          users.forEach((user) => {
```

Comment:
If you're going to use a forEach in a test assertion, we must assert that the array has a length bigger than 0 as we can iterate over an empty array.